src/i18n/en.js

```diff
@@ -1,10 +1,69 @@
 const en = {
     translation: {
         self: {
+            shortDesc: 'An open-source web tool for automatic shoreline mapping and analysis using satellite imagery.',
             abstract: 'A tool integrated with the Google Earth Engine platform capable of offering, in an accessible way, direct access to public data from international satellites, with sophisticated land use analysis capabilities in both space and time. Developed to meet the needs of scientists, managers and students, with a focus on the conservation of the planet Earth\'s natural resources.',
             poweredBy: 'Powered by: Google Earth Engine',
             imageryProvider: 'Imagery Providers: NASA (Landsat) and ESA (Sentinel)'
         },
+        home: {
+            about: {
+                title: 'About',
+            },
+            instructions: {
+                title: 'CASSIE User Guide',
+                text: 'To use CASSIE, you must have a google account registered on the google earth engine platform.',
+                btnEngineSingUp: 'Register',
+                btnManual: 'User Guide',
+                linkManual: 'https://tinyurl.com/cassie-manual-en',
+                btnVideo: 'Instruction Video',
+                linkVideo: 'https://www.youtube.com/watch?v=7q-3xNVq0tY&'
+            },
+            baysqueeze: {
+                title: 'BAYSQUEEZE',
+                text: 'The project seeks to contribute with actions and results for the preservation of our planet, by assuming the task of fulfilling the United Nations Sustainable Development Goals, as well as attending the national needs of knowledge about climate change, in parallel with the goals of the sub-network "Coastal Zones", of the Brazilian Research Network on Global Climate Change (Rede Clima). ',
+                btn: 'Know more about the project'
+            },
+            riscport: {
+                title: 'RISCPORT',
+                text: 'O projeto procura contribuir com ações e resultados para o bem e preservação do nosso planeta, tomando para si, a tarefa de cumprir os objetivos de desenvolvimento sustentável, estabelecidos pela ONU, bem como atender às necessidades nacionais de conhecimento sobre mudanças climáticas, paralelamente aos objetivos da sub-rede "Zonas Costeiras", da Rede Brasileira de Pesquisas sobre Mudanças Climáticas Globais(Rede Clima).',
```

Comment:
Change to:

The main objective of this project is to conduct a climate risk analysis (environmental and economic) and to determine possible adaptation measures, based on the future projection of environmental threats (e.g. storm waves), to increase the resilience of the port sector and adjacent areas in the Babitonga Bay, in an interdisciplinary-multiinstitutional network context.